tensorflow/python/ops/tensor_math_operator_overrides.py

```diff
@@ -60,7 +60,20 @@ def _mod_factory(x, y, name=None):
 
 def _mul_dispatch_factory(x, y, name=None):
   from tensorflow.python.ops import math_ops
-
+  from tensorflow.python.framework import dtypes
+  import tensorflow as tf
```

Comment:
Can we get a shorter, more localized import here? It's not good to write `import tensorflow as tf` inside TF library files as that causes slowdowns during import, now you need to import the entire library before you can import one single file